WebInstall/explainer.md

```diff
@@ -337,21 +340,31 @@ The install capability should not work on *incognito*, and the promise should al
 
 ### Declarative install
 
-An alternate solution to allow installation of web apps is by allowing a new target type of `_install` to the HTML anchor tag.
+An alternate solution is to have a declarative way to install web apps. This can be achieved by allowing a new `target` type of `_install` to the HTML anchor tag. It can also use the [`rel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) attribute to hint to the UA that the url in the link should be installed
```

Comment:
```suggestion
An alternate solution is to have a declarative way to install web apps. This can be achieved by allowing a new `target` type of `_install` to the HTML anchor tag. It can also use the [`rel`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel) attribute to hint to the UA that the url in the link should be installed.
```